compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -298,6 +294,13 @@ class CompilationTests extends ParallelTesting {
     tests.foreach(_.delete())
   }
 
+  @Test def testOptimised: Unit = {
+    val outputDir = defaultOutputDir + "optimised/"
+    compileFilesInDir("../tests/pos", defaultOptimised, outputDir).checkCompile()
+    compileFilesInDir("../tests/run", defaultOptimised, outputDir).checkRuns()
+    compileShallowFilesInDir("../tests/neg", defaultOptimised, outputDir).checkExpectedErrors()
```

Comment:
Should we move each test in its corresponding test group (i.e. `compilePos`, `compileNeg`, `runAll`) to run them in parallel or should we keep them together for clarity?

Reply:
I would prefer to keep the separated for now because I going to start modifying stuff in the local optimizations. It will be clearer to see where the failures occurred if they happen. There are enough tests in each one to take advantage of parallelism, we can improve it later.